staging/src/k8s.io/apiserver/pkg/util/flag/ciphersuites_flag.go

```diff
@@ -54,11 +54,19 @@ func TLSCipherSuites(cipherNames []string) ([]uint16, error) {
 	}
 	ciphersIntSlice := make([]uint16, 0)
 	for _, cipher := range cipherNames {
-		intValue, ok := ciphers[cipher]
-		if !ok {
-			return nil, fmt.Errorf("Cipher suite %s not supported or doesn't exist", cipher)
+		intValue, err := TLSCipherSuite(cipher)
+		if err != nil {
+			return nil, err
 		}
 		ciphersIntSlice = append(ciphersIntSlice, intValue)
 	}
 	return ciphersIntSlice, nil
 }
+
+func TLSCipherSuite(cipherName string) (uint16, error) {
```

Comment:
this should be `TLSVersion(tlsVersion string) (uint16, error)` that deals with these constants:

```
        VersionSSL30 = 0x0300
        VersionTLS10 = 0x0301
        VersionTLS11 = 0x0302
        VersionTLS12 = 0x0303
```